cmd/apiserver/app/server/run_server.go

```diff
@@ -183,7 +183,7 @@ func (c checkEtcdConnectable) Check(_ *http.Request) error {
 	serverReachable, err := preflight.EtcdConnection{ServerList: c.ServerList}.CheckEtcdServers()
 
 	if err != nil {
-		glog.Error("etcd checker failed with err: %v", err)
+		glog.Errorf("etcd checker failed with err: %v", err)
```

Comment:
I haven't looked at the details of the call chain, but shouldn't it be either logging OR returning that error? That caller will probably also log this so I'll end up in the logs twice, causing confusion. Why not `return fmt.Errorf(....)`?